gum/backend-x86/gumstalker-x86.c

```diff
@@ -78,6 +78,7 @@ struct _GumStalker
   GumExceptor * exceptor;
   gpointer user32_start, user32_end;
   gpointer ki_user_callback_dispatcher_impl;
+  gpointer wow64transition_address;
```

Comment:
I decided to use `wow64transition_address` on purpose because the symbol is called exactly "Wow64Transition". But I have no strong opinion about that.